src/test/regress/expected/alter_distributed_table.out

```diff
@@ -537,23 +537,16 @@ SELECT create_distributed_table('referenced_dist_table', 'a', colocate_with:='no
 (1 row)
 
 SELECT create_reference_table('referenced_ref_table');
- create_reference_table
----------------------------------------------------------------------
-
-(1 row)
-
+ERROR:  cannot create foreign key constraint since foreign keys from reference tables and local tables to distributed tables are not supported
```

Comment:
Hmm, why is that failing?

Reply:
This is expected.

But still we should create the foreign keys after create_distributed_table and create_refence_table calls, and then create the foreign keys